modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppFooter.js

```diff
@@ -41,7 +41,9 @@ class App extends Component {
       arrows: false,
       variableWidth: false,
       centerMode: true,
-      centerPadding: '0px'
+      centerPadding: '0px',
+      autoplay:true,
+      autoplaySpeed:5000,
```

Comment:
Could you please add blank symbol after colons?

Reply:
@duozersk  Thank you for let me know standard of coding.